modular_zubbers/code/datums/quirks/positive_quirks/venemous_bite/venemous_bite.dm

```diff
@@ -0,0 +1,124 @@
+#define VENEMOUS_BITE_DAMAGE 5
+#define VENEMOUS_BITE_WOUND_CHANCE 20
+#define VENEMOUS_BITE_WOUND_BONUS 40
```

Comment:
The wounding is slightly worrying. But it would need further testing since I'm not 100 percent present with all the wounding code and how potent these values are.

Reply:
a level one puncture wound is negligible and usually clots on its own, and exists to introduce a bit of extra downside to using the medicines